modules/monitoring-configuring-a-local-persistent-volume-claim.adoc

```diff
@@ -40,12 +38,12 @@ data:
           storageClassName: *_storage class_*
           resources:
             requests:
-              storage: *_40Gi_*
+              storage: *_amount of storage_*
```

Comment:
We aren't using this styling for our YAML annotations (https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference).

Reply:
Thank you, implemented this in #16978.